src/dstr-binding-async-iteration/ary-ptrn-elision-step-err.case

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Elision advances iterator and forwards abrupt completions
+template: error
+info: |
+    13.3.3.6 Runtime Semantics: IteratorBindingInitialization
+
+    ArrayBindingPattern : [ Elision ]
+
+    1. Return the result of performing
+       IteratorDestructuringAssignmentEvaluation of Elision with iteratorRecord
+       as the argument.
+
+    12.14.5.3 Runtime Semantics: IteratorDestructuringAssignmentEvaluation
+
+    Elision : ,
+
+    1. If iteratorRecord.[[done]] is false, then
+       a. Let next be IteratorStep(iteratorRecord.[[iterator]]).
+       b. If next is an abrupt completion, set iteratorRecord.[[done]] to true.
+       c. ReturnIfAbrupt(next).
+features: [generator]
+---*/
+
+//- setup
+var following = 0;
+var iter =function* () {
+  throw new Test262Error();
+  following += 1;
```

Comment:
this is a small issue now that we can't verify the value of `following` in the given error/* templates.

My suggestion is to change the templates to support assertions in the rejection body, so a file in the `error` folder, like `error/for-await-of-async-func-const.template` would end with:


```js
fn()
  .then(_ => {
    throw new Test262Error("Expected async function to reject, but resolved.");
  }, ({ constructor }) => {
    assert.sameValue(constructor, /*{ error }*/);
    /*{ rejectBody }*/
  })
  .then($DONE, $DONE);
```

and you can finally add to this case file:

```js
//- rejectBody
assert.sameValue(following, 0, "iterator is properly closed");
```